.travis.yml

```diff
@@ -1,11 +1,15 @@
 sudo: false
 language: node_js
 node_js:
-  - '0.12'
-  - '0.10'
-  - iojs
+  - 4
+  - "0.10"
+  - "stable"
+
+before_install:
+  - "npm config set progress=false"
```

Comment:
@novemberborn I think it does speed up install a bit, it definitely does locally... I don't think there's harm in having it on CI since like you said Travis doesn't render one anyway, but if there's even a slight benefit then it's worth it imho.